.travis.yml

```diff
@@ -17,6 +18,8 @@ env:
   - DJANGO=1.10
   - DJANGO=1.11
   - DJANGO=2.0
+  - DJANGO=2.1
+  - DJANGO=2.2
 
 install: pip install "virtualenv<14.0.0" tox-travis
```

Comment:
The error message on Travis CI suggests that the installation of `tox-travis` is pulling a version of `tox` that is too new. Could you try to add `"tox<3"` to this command? e.g.
```yaml
install: pip install "virtualenv<14.0.0" "tox<3" tox-travis
```
It could be that this fixes the current error. Then, maybe, some others will pop up...